src/main/webapp/js/script.js

```diff
@@ -76,3 +77,93 @@ function showOrHideNotes(version) {
     $('div.setup-panel div a.btn-success').trigger('click');
 });
 
+
+//Fetch data from API endpoint
+ 
+// Fetch for landing page
+let doms = ["web", "android", "games", "ios"]
+
+async function getPlatforms() {
+    const response = await fetch('/v1/platforms');
+    const platforms = await response.json();
+  
+    count = 0;
+    for (let plat in platforms){
+       if (plat == doms[count]){
+        const divElement = document.getElementById(plat); 
+        const textNode = document.createTextNode(platforms[plat]); 
+        divElement.appendChild(textNode);
+       }
+       count+=1;
+    }
+}
+
+//fetch release pages
+async function getReleases() {
+   const response = await fetch('/v1/platforms/android/releases');
```

Comment:
I think you should add a parameter of platform and then in the fetch do `v1/platforms/${platform}/releases`